wrappers/tensorflow2.2.0_wrapper.py

```diff
@@ -0,0 +1,55 @@
+import pandas
+import tensorflow
+
+import tensorflow_hub
+
+
+class TensorFlowWrapper:
+    """
+    Wrapper object for TensorFlow graph and helps use it.
+    """
+
+    def __init__(self, embedding_layer_hub_name: str) -> None:
+        g = tensorflow.Graph()
+        with g.as_default():
+            # Import the Universal Sentence Encoder's TF Hub module
+            embedding_layer = tensorflow_hub.Module(embedding_layer_hub_name)
+
+            self._sts_input1 = tensorflow.compat.v1.placeholder(tensorflow.string, shape=None)
```

Comment:
use tf.disable_v2_behavior() if you'd like to use tf1 behavior. 
I suggest moving to tensorflow v2 behavior. Replace compat.v1.placeholder with their match in tf2 such as:
placeholder become Variable or keras.layers.Model.

See full guide here: https://www.tensorflow.org/guide/migrate